package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "node-onvif",
-  "version": "0.1.7",
+  "version": "0.1.8",
```

A feature would bump the semver to 0.2.0
```suggestion
  "version": "0.2.0",
```